ddtrace/tracer.py

```diff
@@ -118,15 +91,15 @@ def __init__(self, url=None, dogstatsd_url=DEFAULT_DOGSTATSD_URL):
 
         uds_path = None
         https = None
-        hostname = self.DEFAULT_HOSTNAME
-        port = self.DEFAULT_PORT
+        hostname = environ.get("DD_AGENT_HOST", environ.get("DATADOG_TRACE_AGENT_HOSTNAME")) or self.DEFAULT_HOSTNAME
```

Comment:
the hostname/port changes are unrelated to this patch and so they can be moved to #1848 or a separate PR

Reply:
👍🏽